docs/markdown/radial-chart.md

```diff
@@ -1,89 +1,124 @@
 # Radial chart
 
-`RadialChart` is responsible for creating pie and donut charts.
-Note: currently radial chart _does not_ support series and it is not planned in the nearest future.
+`RadialChart` is responsible for creating pie and donut charts. While this kind of chart is easy to overlook as insignificant, intentionally confusing, or almost always replaceable with a treemap; they can be useful for quickly showing small groups. People don't understand angles very well (such is our biology), but over the last hundred years we have seen a lot of pie charts! This has caused us to become intimately familiar with them.
```

Comment:
Great intro! Would be really cool to link to resources or articles explaining why/how we know (biologically) that we don't understand angles well.